README.md

```diff
@@ -1 +1,72 @@
-# awesomeBooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> A Page to add (and remove) books from a list
+
+## Built With
+
+- HTML, CSS, JS
+- Github
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- Code Editor (e.g. VS Code, Atom)
+- Web browser 
+
+### Setup
+
+ **Step 1: Clone this repo locally using git**
+- Create a folder in your local machine
+- Open your console in your folder and run this command: `git clone https://github.com/Daf98/awesomeBooks.git`
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+ **Step 2: Run in your project directory**
+- Open your console in your folder and run the nex command `. code` to open your vscode editor with awesomeBooks project.
+- Then in index.html you can open with live server.
+
+## Authors
+
+👤 **Dafne Azzolina**
+
+- GitHub: [@Daf98](https://github.com/Daf98)
+- Twitter: [@dafne_azzolina](https://twitter.com/dafne_azzolina)
+- LinkedIn: [dafne-azzolina](https://www.linkedin.com/in/dafne-azzolina/)
+
+👤 **Dino Quispe**
+
+- GitHub: [@dqarias](https://github.com/dqarias)
+- Twitter: [@DinoRonald7](https://twitter.com/DinoRonald7?t=Zanx9DXMEG9C_PNF3woZFg&s=08)
+- LinkedIn: [Dino Quispe](https://www.linkedin.com/in/dino-ronald-quispe-arias-8ba72174/)
+
+👤  **Owoeye Olaoluwa (Olamarx)**
+
+- GitHub: [@githubhandle](https://github.com/Olamarx)
+- Twitter: [@twitterhandle](https://twitter.com/Owoeye0laoluwa)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/olaoluwa-owoeye-617702162/)
+- Facebook: [Facebook](https://web.facebook.com/olaoluwa.owoeye.39)
```

Comment:
OPTIONAL.

This being a pair-programming project, all members should have contributed to this project. No commits from the 3rd participant are visible and all along, I thought you were only 2 members for this. This would have been a good reason for invalidation. Otherwise, always make sure that all participants contribute to the project.

Reply:
@mke2111 Participant removed

Reply:
No worries. Your project will be approved manually from your dashboard. This may take a few hours to reflect. In the meantime,  you can merge and proceed with the next milestones.